cmd/buildkitd/constants_windows.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+const (
+	defaultContainerdAddress = "//./pipe/containerd-containerd"
+	defaultCNIBinDir         = "C:/ProgramData/buildkitd/bin"
+	defaultCNIConfigPath     = "C:/ProgramData/buildkitd/cni.json"
```

Comment:
Would this have to use `%PROGRAMDATA%` ?

Reply:
These can be anywhere on the system. Couldn't decide where to put them. Buildkit has a default config dir, and I thought it would make sense to put them there. 

I think we could define these in  https://github.com/moby/buildkit/tree/master/util/appdefaults to keep things consistent.

I don't feel strongly about the default location for these. Any suggestions are welcome.

Reply:
Yeah, probably one of the BuildKit maintainers would be able to tell the best spot for the consts; was mostly wondering if the path to (either "Program Files" or "ProgramData") should be hard-coded, or if it should pick up the `%PROGRAMDATA%` or `%PROGRAMFILES%` env-var.

Reply:
Definitely not hard coded. Will move them to the appdefaults package and join them to the default config dir wgich expands ```APPDATA```

Reply:
Should be `%PROGRAMDATA%` for system-wide data (services, drivers config, ...) or user's folder `%APPDATA%`.

Reply:
The ```appdefaults``` package uses ```%PROGRAMDATA%``` so that seems to be the perfect place. Will change it shortly.

Reply:
The CNI  constants were moved to the ```appdefaults``` package.